angular/src/app/utils/cbor.js

```diff
@@ -0,0 +1,382 @@
+(function(global, undefined) { "use strict";
```

Comment:
Does WebAuthn rely on CBOR? Where is this implementation taken from? Shouldn't rely on an existing CBOR implementation instead?